blaze/compute/tests/test_sql_compute.py

```diff
@@ -348,11 +348,11 @@ def test_by_summary_clean():
 
 
 def test_by_summary_single_column():
-    expr = by(t.name, n=t.name.count(), biggest=t.name.max())
+    expr = by(t.name, n=t.name.count(), biggest=t.amount.max())
```

Comment:
If you have an issue with computing the maximum of a string then I suggest changing all of the columns from `t.name` to `t.amount.`